apps/old/widget/components/Navbar.jsx

```diff
@@ -242,11 +245,18 @@ function Navbar(props) {
   };
 
   const TestBtn = () => {
+    const { networkId } = context;
+
+    const isTestnet = networkId === "testnet";
     return (
       <a
         target="_blank"
-        className="grey-btn"
-        href="https://test.nearbuilders.org"
+        className={`grey-btn ${isTestnet ? "testnet" : ""}`}
+        href={
+          isTestnet
+            ? "https://www.nearbuilders.org/"
+            : "https://test.nearbuilders.org/"
+        }
       >
         <img
           src="https://ipfs.near.social/ipfs/bafkreieud33bpqibciatt6uwqju4r3xk7jwy3bunfgiz35oiwyiapbcjbq"
```

Comment:
You can use this icon `<i className="bi bi-wifi"></i>` and follow the new design form the figma file
 - [Figma](https://www.figma.com/design/O9VBPoKQH80YLndvcbj3Zf/[WorkSpace]-HomePage-2.0?node-id=0-1&t=Mkv1CWCN2WsaN7RN-0)
 - Image
 
![Screen Shot 2024-05-21 at 4 12 49 PM](https://github.com/NEARBuilders/gateway/assets/65053264/1620a754-fd17-46ff-91d5-234ab163c785)